episodes/17-live.md

```diff
@@ -90,9 +106,9 @@ Some challenges are:
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-Live coding fits well into the practice-feedback model we have been discussing - by providing
-learners with continuous opportunities for practice (every time they type in a line of code)
-and continuous feedback (their code either works or fails with an error message). It is
+Live demos fits well into the practice-feedback model we have been discussing - by providing
```

Comment:
```suggestion
Live demonstration fits well into the practice-feedback model we have been discussing - by providing
```
i think using the full work is better for non native english speakers, but would accept the singular of the shortcut